appium/webdriver/extensions/android/nativekey.py

```diff
@@ -1012,17 +1012,17 @@ def is_gamepad_button(code):
     confirm_buttons = [DPAD_CENTER, ENTER, SPACE, NUMPAD_ENTER]
 
     @staticmethod
-    def is_confirm_key(code):
```

Comment:
oops, you didn't have to update method names.
Just follow java-client's naming.

Reply:
Sure, Thanks.